test/test_foreach.py

```diff
@@ -1054,25 +1054,32 @@ def test_addcdiv(self, device, dtype):
                 return
         self._test_pointwise_op(device, dtype, torch._foreach_addcdiv, torch._foreach_addcdiv_, torch.addcdiv)
 
-    @dtypes(*torch.testing.get_all_dtypes(include_bfloat16=False, include_bool=False, include_complex=False))
+    @dtypes(*torch.testing.get_all_dtypes())
```

Comment:
Is there a type promotion test for foreach_maximum / foreach_minimum, or is that out of the scope of this PR?

Reply:
Type promotion will be tested in upcoming PRs once we push different dtype behavior from error to slow path.